guides/source/wishlists.md

```diff
@@ -0,0 +1,1545 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON <https://guides.rubyonrails.org>.**
+
+Wishlists
+=========
+
+This guide covers adding Wishlists to the store e-commerce application in the
+[Getting Started Guide](getting_started.html)). We will use the code from the
+[Sign up and Settings Guide](sign_up_and_settings.html) as a starting place.
+
+After reading this guide, you will know how to:
+
+* Add wishlists
+* Use counter caches
+* Add friendly URLs
+* Filter records
+
+--------------------------------------------------------------------------------
+
+Introduction
+------------
+
+E-commerce stores often have wishlists for sharing products. Customers can use
+wishlists to keep track of products they'd like to buy or share them with
+friends and family for gift ideas.
+
+Let's get started!
+
+Wishlist Models
+---------------
+
+Our e-commerce store has Products and Users that we built in the previous
+tutorials. These are the foundations we need to build Wishlists. Each wishlist
+belongs to a user and contains a list of Products.
+
+Let's start by creating the Wishlist model.
+
+```bash
+$ bin/rails generate model Wishlist user:belongs_to name products_count:integer
+```
+
+This model has 3 attributes:
+
+- `user:belongs_to` which associates the Wishlist with the `User` record who
+  owns it
+- `name` which we'll also use for friendly URLs
+- `products_count` for the counter cache we'll add to count how many products
```

Comment:
I think we never introduce counter caches (maybe in previous tutorials?). Anyway, a link to the docs wouldn't hurt.
```suggestion
- `products_count` for the [counter cache](https://guides.rubyonrails.org/association_basics.html#counter-cache) we'll add to count how many products
```